### PR TITLE
COM-54 PECG: DATA COLLECTION FORM OF PLWHIV UNDER ARVs DISAGGREGATE B…

### PR DESCRIPTION
**Pull request by prashantsable1991**
Added pecg template and logic to show records in template.

### CODE REVIEW

**Comment by lauravignoli** [comment]
Hi @prashantsable1991 great work so far (still busy with the review).

I have a small comment, can you please bold Region: and Reporting Period: ?

**Comment by lauravignoli** [comment]
Also, please use years (with s) when there's more than 1 year. And add a space after < where needed:
years

File: openmrs/apps/reports/PECG_REPORT/sql/data_collection.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+SELECT 'PATIENT DATA COLLECTION' AS '-'  ,
+            PECG_DATA_COLLECTION('#startDate#','#endDate#', 0, 1, 0, 'M') AS '< 1 year M', 
+            PECG_DATA_COLLECTION('#startDate#','#endDate#', 0, 1, 0, 'F') AS '< 1 year F',
+            PECG_DATA_COLLECTION('#startDate#','#endDate#', 1, 2, 1, 'M') AS '1 - <2 year M',
```
Comment (lauravignoli) [inline openmrs/apps/reports/PECG_REPORT/sql/data_collection.sql]: @prashantsable1991 includeEndAge should be 0 here

**Comment by prashantsable1991** [comment]
Thanks for review @lauravignoli I have pushed the required review changes please review once.

**Pull request merged**